paper-tabs.html

```diff
@@ -377,6 +378,39 @@
         }, 10);
       },
 
+      _onIronItemsChanged: function(event) {
```

Comment:
I still need to call [this function that it overrides](https://github.com/PolymerElements/iron-menu-behavior/blob/0ba8b7d/iron-menu-behavior.html#L195) and (maybe?) remove the listener above since it has the same signature.